templates/base.html

```diff
@@ -0,0 +1,181 @@
+    <meta name="description" content="" />
+    <meta name="author" content="" />
+    <title>Education Policy Assistant</title>
+    {% load static %}
+    {% load get_user_context %}
+
+    <link href="{% static 'css/styles.css' %}" rel="stylesheet" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/js/all.min.js"
+        crossorigin="anonymous"></script>
+</head>
+
+{% _get_user_context request as user_context %}
+
+<body class="sb-nav-fixed">
+    <nav class="sb-topnav navbar navbar-expand navbar-dark bg-dark">
+        <a class="navbar-brand" href="index.html">EMIS</a>
+        <button class="btn btn-link btn-sm order-1 order-lg-0" id="sidebarToggle" href="#"><i
+                class="fas fa-bars"></i></button>
+        <!-- Navbar Search-->
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+            {% if 'apps_permission' in user_context and 'school' in user_context.apps_permission %}
+            <li class="nav-item">
+                <a class="nav-link {% if 'school' in request.path %} active {% endif %}" href="#">Schools</a>
+            </li>
+            {% endif %}
+            {% if 'apps_permission' in user_context and 'welfare' in user_context.apps_permission %}
+            <li class="nav-item">
+                <a class="nav-link {% if 'welfare' in request.path %} active {% endif %}" href="#">Welfare</a>
+            </li>
+            {% endif %}
+            {% if 'apps_permission' in user_context and 'historical' in user_context.apps_permission %}
+            <li class="nav-item">
+                <a class="nav-link {% if 'historical' in request.path %} active {% endif %}"
+                    href="/historical">Historical</a>
+            </li>
+            {% endif %}
+            {% if 'apps_permission' in user_context and 'authentication' in user_context.apps_permission %}
+            <li class="nav-item">
+                <a class="nav-link {% if 'auth' in request.path %} active {% endif %}"
+                    href="/auth/sysadmin/users">Authentication</a>
+            </li>
+            {% endif %}
+        </ul>
+        <form class="d-none d-md-inline-block form-inline ml-auto mr-0 mr-md-3 my-2 my-md-0">
+        </form>
+        <!-- Navbar-->
+        <ul class="navbar-nav ml-auto ml-md-0">
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" id="userDropdown" href="#" role="button" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false"><i class="fas fa-user fa-fw"></i></a>
+                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="userDropdown">
+                    <a class="dropdown-item" href="/auth/logout">Logout</a>
+                </div>
+            </li>
+        </ul>
+    </nav>
+    <div id="layoutSidenav">
+        <div id="layoutSidenav_nav">
+            <nav class="sb-sidenav accordion sb-sidenav-light" id="sidenavAccordion">
```

Comment:
Is there an option on the accordions to leave them open after they're clicked? This is a small change but I think it could reduce clicks and improve search speed if items stayed expanded so you can see sub-menu items all at once.